ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/route53/private-hosted-zone/provision/cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone-provision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone-provision
+  steps:
+    - ref: ipi-conf
+    - ref: aws-provision-vpc-shared
+    - ref: aws-provision-bastionhost
```

Comment:
I do not think bastion host is necessary for this configuration.

Reply:
@mhanss bastion host is not required, since it's a simple connected IPI cluster.

Reply:
Done